contrib/scrooge/tests/python/pants_test/contrib/scrooge/tasks/test_thrift_linter_integration.py

```diff
@@ -18,55 +18,55 @@ def test_good(self):
     cmd = ['thrift-linter', self.thrift_test_target('good-thrift')]
     pants_run = self.run_pants(cmd)
     self.assert_success(pants_run)
-    self.assertFalse('Lint errors found!' in pants_run.stdout_data)
+    self.assertFalse('LINT-ERROR' in pants_run.stdout_data)
```

Comment:
assert_success already checks the return code.
I believe the original intention of this extra check is for warnings outputs.